english/movie-ripoffs/hogwarts/burn-marshmallows

```diff
@@ -0,0 +1,5 @@
+It works! It actually works! You are a wizard!
+
+This thought is a great comfort to you as the burning, melting marshmallow-structure collapses around you. So is the taste of roasted marshmallow. Until you suffocate and die. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.